sdk/src/fee.rs

```diff
@@ -31,6 +31,19 @@ pub struct FeeStructure {
     pub compute_fee_bins: Vec<FeeBin>,
 }
 
+/// Return type of calculate_fee(...)
+#[derive(Debug, Default, Clone, Eq, PartialEq)]
+pub struct FeeDetails {
+    pub transaction_fee: u64,
+    pub prioritization_fee: u64,
```

Comment:
doesn't seem like these are ever (de)structured outside the mod, so no sense exposing them to public api
```suggestion
    transaction_fee: u64,
    prioritization_fee: u64,
```